File: src/main/java/com/hobojoe/enderore/proxy/ClientProxy.java
package com.hobojoe.enderore.proxy;

import com.hobojoe.enderore.EnderOre;
import net.minecraft.client.renderer.block.model.ModelResourceLocation;
import net.minecraft.item.Item;
import net.minecraftforge.client.model.ModelLoader;

/**
 * Created by Joseph on 11/18/2016.
 */
public class ClientProxy extends CommonProxy {

    @Override
    public void registerItemRenderer(Item item, int meta, String id) {
        ModelLoader.setCustomModelResourceLocation(item, meta, new ModelResourceLocation(EnderOre.MODID + ":" + id, "inventory"));
    }
}

File: src/main/java/com/hobojoe/enderore/proxy/ClientProxy.kt
package com.hobojoe.enderore.proxy


import com.hobojoe.enderore.block.ModBlocks
import com.hobojoe.enderore.item.ModItems
import net.minecraftforge.client.event.ModelRegistryEvent
import net.minecraftforge.fml.common.Mod
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent

/**
 * Created by Joseph on 11/18/2016.
 */

@Mod.EventBusSubscriber
class ClientProxy : CommonProxy() {

    companion object {
        @JvmStatic
        @SubscribeEvent
        fun registerModels(event: ModelRegistryEvent) {
            ModBlocks.initModels()
            ModItems.initModels()
        }
    }
}
